lib/certificate.js

```diff
@@ -39,7 +39,16 @@ class Certificate {
   async has (dir) {
     let subject = await this.getSubject(this.pemFilename);
     let trustStore = new TrustStore(dir);
-    return await trustStore.hasRecords(subject);
+
```

Comment:
would it be possible to add better docstring description to this method and also other methods in this class?